website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -80,6 +80,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the CLI or the runs API.                                                                                                                                                                                    |
 | `data.attributes.terraform-version`           | string  | (special) | The version of Terraform to use for this workspace. This can be either an exact version or a [version constraint](/language/expressions/version-constraints) (like `~> 1.0.0`); if you specify a constraint, the workspace will always use the newest release that meets that constraint. If omitted when creating a workspace, this defaults to the latest released version.                                                                                                                                   |
 | `data.attributes.trigger-prefixes`            | array   | `[]`      | List of repository-root-relative paths which should be tracked for changes, in addition to the working directory.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.trigger-patterns`            | array   | `[]`      | List of repository-root-relative glob patterns which describe files tracked for changes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
What do we mean by "repository root relative" here? If you give me some more details I will think about how we can make sure it's clear to users :)

Reply:
That means the provided triggers always start from the root of a repository.
So if someone inputs trigger `/test/submodule`, that trigger will be prepended with wherever the repository is located -> e.g. `/Downloads/repository/test/submodule`.

Reply:
ahhh okay okay - i made a suggestion to clarify this a bit. See what you think!